.nvmrc

```diff
@@ -0,0 +1 @@
+16
```

Comment:
Sets a default node version to use in the project. NVM makes different node versions easier to manage. Community Solid Server expects Node 16. It seems to run on Node 18 fine, but doesn't officially support it.